### DISCUSSION TITLE
/sale/offers/unfilled-parameters zwraca tylko listę z ofertami które są aktywne?

### DISCUSSION THREAD

**Discussion by bugproof** [allegro/allegro-api#4431] [Suggestions - rejected]
https://github.com/allegro/allegro-api/issues/4147

**Comment by MaciejFrackowiak** [comment]
Tak, zasób co do zasady zwraca tylko aktywne oferty. Ich odświeżenie odbywa się raz dziennie, zatem mogą się też pojawić tam oferty zakończone w ciągu ostatnich 24 godzin.

**Comment by bugproof** [comment]
Dobrze jakby był jakiś filtr który pozwalałby na zwracanie też nieaktywnych ofert, bo są np. takie oferty których nie można ponownie wystawić bo brakuje jakiegoś parametru a można go automatycznie wypełnić np. na podstawie opisu lub tytułu oferty lub nawet na podstawie innego parametru

Chyba obecnie będę musiał zdać się na ręczne sprawdzanie czy dany parametr jest wypełniony, czy nie.

**Comment by MaciejFrackowiak** [comment]
Dziękuję za sugestię. Zgodnie z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337) przeniosłem wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions).

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
